realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -342,4 +369,49 @@ private static void copyAssetFileIfNeeded(RealmConfiguration configuration) {
             }
         }
     }
+
+    /**
+     * Finds an entry for specified schema version in the array.
+     *
+     * @param array target array of schema cache.
+     * @param schemaVersion requested version of the schema.
+     * @return {@link ColumnIndices} instance for specified schema version. {@code null} if not found.
+     */
+    public static ColumnIndices find(ColumnIndices[] array, long schemaVersion) {
+        for (int i = array.length - 1; 0 <= i; i--) {
```

Comment:
I think foreach loop would be enough? and easy to write :)
especially after the array get wrapped, the latest cache might be at index 0.
Anyway, minor thing.